services/conda/conda-downloads.service.js

```diff
@@ -1,27 +1,46 @@
 'use strict'
 
-const LegacyService = require('../legacy-service')
+const BaseCondaService = require('./conda-base')
+const { metric } = require('../../lib/text-formatters')
+const { downloadCount } = require('../../lib/color-formatters')
 
-module.exports = class CondaDownloads extends LegacyService {
+module.exports = class CondaDownloads extends BaseCondaService {
   static get category() {
     return 'downloads'
   }
 
   static get route() {
     return {
       base: 'conda',
+      pattern: ':which(d|dn)/:channel/:pkg',
```

Comment:
I'm not sure if we actually need the `(d|dn)` or `(p|pn)` params. The behaviour of both the `p`/`d` and `pn`/`dn` versions is tested, but there are no examples in the front-end for `d` or `p` :thinking:

Reply:
It seems the only difference between `xn` and `x` is in the label, and the developer picks one or the other based on whether they're publishing to the default channel. (I wonder if we can't get that information from the API.) It's honestly a bit confusing to me; wouldn't mind getting clarification from some developers who use it. Not necessary for this PR though.